contributors/guide/issue-triage.md

```diff
@@ -4,57 +4,233 @@ weight: 10
 
-Most people can leave comments and open issues. They don't have the ability to
-set labels, change milestones and close other people's issues. For that we use
-a bot to manage labelling and triaging. The bot has a set of
-[commands and permissions](https://go.k8s.io/bot-commands)
-and this document will cover the basic ones.
+### Conducting Searches
+GitHub allows you to filter out types of issues and pull requests, which helps you discover items in need of triaging. This table includes some predetermined searches for convenience:
 
-## Determine if it's a support request
+|  Search | What it sorts  |
+|---|---|
+| [created-asc](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-asc)  | untriaged issues by age |
+|  [needs-sig](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds-sig) | issues that need to be assigned to a SIG  |
+| [`is:open is:issue`](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue)   | Newest incoming issues  |
+| [comments-desc](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+sort%3Acomments-desc)   | busiest untriaged issues, sorted by # of comments  |
+| [comments-asc](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+sort%3Acomments-asc)   | Issues that need more attention, based on # of comments  |
 
-Sometimes users ask for support requests in issues; these are usually requests
-from people who need help configuring some aspect of Kubernetes. These issues
-should be labeled with `triage/support`, directed to our support structures
-(see below) and then closed. Also, if the issue is clearly abandoned or in the
-wrong place, it should be closed. Keep in mind that only issue reporters,
-assignees and component organization members can close issues. If you do not
-have such privilege, just comment your findings. Otherwise, first `/assign`
-issue to yourself and then `/close`.
+We suggest preparing your triage by filtering out the oldest, unlabelled issues and/or pull requests first.
+
+## Step Two: Triage Issues by Type
+Use [these labels](https://github.com/kubernetes/kubernetes/labels?utf8=%E2%9C%93&q=triage%2F+kind%2Fsupport) to find open issues that can be quickly closed. A triage engineer can add the appropriate labels.
+
+Depending on your permissions, either close or comment on any issues that are identified as support requests, duplicates, or not-reproducible bugs, or that lack enough information from the reporter.
+ 
+### Support Requests
+Some people mistakenly use GitHub issues to file support requests. Usually they're asking for help configuring some aspect of Kubernetes. Here's how to handle these:
+* First, direct the person to discuss the issue on [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes) and visit the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/)
```

Comment:
We should point to this section "Support Requests: Channels" vs only mentioning Stack Overflow and troubleshooting guide (i.e. use something like, direct the person to use support request channels)